plenario/api/validator.py

```diff
@@ -1,15 +1,27 @@
+import json
+
 from collections import namedtuple
 from datetime import datetime, timedelta
 from dateutil import parser
 from marshmallow import fields, Schema
-from marshmallow.validate import Range, Length, OneOf
-from sqlalchemy.exc import DatabaseError, ProgrammingError
+from marshmallow.validate import Range, Length, OneOf, ValidationError
+from sqlalchemy.exc import DatabaseError, ProgrammingError, NoSuchTableError
 
 from plenario.api.common import extract_first_geometry_fragment, make_fragment_str
+from plenario.api.condition_builder import field_ops
 from plenario.database import session
 from plenario.models import MetaTable, ShapeMetadata
 
 
+def validate_many_datasets(list_of_datasets):
```

Comment:
dataset_name__in now get's checked for every table it holds. Originally it was just checked to be a list